Documentation/ceph-csi-drivers.md

```diff
@@ -138,3 +138,24 @@ csi_liveness 1
 
 Check the [monitoring doc](ceph-monitoring.md) to see how to integrate CSI
 liveness and grpc metrics into ceph monitoring.
+
+### Dynamically Expand Volume
+
+#### Prerequisite
+
+* For filesystem resize to be supported for your Kubernetes cluster, the
```

Comment:
Is this checked anywhere in the code?
We should probably check and log messages depending on if the functionality is available or not.

Reply:
do you want to log it in rook operator?

Reply:
Yes please, that 'd be helpful.